assignment-01-cat/Cat.cpp

```diff
@@ -1,15 +1,69 @@
 #include "Cat.h"
+#include <cstring>
 
-Cat::Cat(const char* _name) {
+Cat::Cat(const char* _name)
```

Comment:
```suggestion
Cat::Cat(const char* _name): Cat(_name, -1, false)
```
이런 식으로 다른 생성자를 호출해주면 매 생성자마다 문자열복사를 해주지 않아도 돼요!
코드 중복을 줄여줍시다.